src/typeDefs/types.ts

```diff
@@ -92,6 +92,15 @@ export const types = gql`
     endDate: Date!
   }
 
+  type UpdateAdvertisementPayload {
+    _id: ID
+    name: String!
+    link: String!
+    type: String!
```

Comment:
this should be an enum too, just like it's an enum in the database model, use a graphql enum named `AdvertisementType` and make use of SCREAMING_SNAKE_CASE(if needed to seperate words) for different instances of the enum